integrations/gitlab/gitlab_integration/git_integration.py

```diff
@@ -89,6 +92,12 @@ def _search(self, data: Dict[str, Any], pattern: str) -> Any:
 class GitlabPortAppConfig(PortAppConfig):
     spec_path: str | List[str] = Field(alias="specPath", default="**/port.yml")
     branch: str = "main"
+    filter_owned_projects: bool | None = Field(
+        alias="filterOwnedProjects", default=True
+    )
+    project_visibility_filter: str | None = Field(
+        alias="projectVisibilityFilter", default=None
+    )
```

Comment:
i think the format should match
decide if you want to start the var with`filter` or end it with it